src/ogma/features/styles.ts

```diff
@@ -375,7 +375,7 @@ export class StylesViz {
       name: 'filtered',
       nodeAttributes: {
         opacity: FILTER_OPACITY,
-        layer: -1,
+        layer: 0,
```

Comment:
set the layer of the filtered nodes to zero and the layer of the virtual node to -1 to make them visible